### ISSUE TITLE
Re-run menu entry didn't work

### ISSUE THREAD

**Issue by jay** [cirruslabs/cirrus-ci-web#396]
There's a rerun button that appears on my tasks page with a drop down menu that has two entries, re-run and re-run in terminal. I clicked the re-run entry a few times but nothing happened. Then I clicked the actual re-run button and it worked. 

Here's an animated GIF from [a task](https://cirrus-ci.com/task/4508011075993600) that I cannot restart unless I click the button:

![re-run doesn't work](https://user-images.githubusercontent.com/965580/136113677-36f824d2-95e5-4e40-aed8-8fd932cadf6f.gif)

**Comment by fkorotkov** [comment]
Agreed. Current UX is confusing. Just to confirm, you'd expect a click in the dropdown to actually re-run a task without a need for an extra click on the button itself?

**Comment by jay** [comment]
> Just to confirm, you'd expect a click in the dropdown to actually re-run a task without a need for an extra click on the button itself?

Yes, I would expect clicking on the menu entry to re run the task.
